server/controllers/bot.ctrl.js

```diff
@@ -0,0 +1,80 @@
+const bunyan = require('bunyan');
+// const path = require('path');
+
+// const Receive = require(path.join(__dirname, '../services/receive'));
+
+const log = bunyan.createLogger({ name: 'messengerBot' });
+
+const users = {};
+
+module.exports = {
+  // Get Webhook
+  getWebhook: (req, res) => {
+    const { VERIFY_TOKEN } = process.env;
+
+    // Parse params from the webhook verification request
+    // TODO: Try if destructuring would work
+    const mode = req.query['hub.mode'];
+    const token = req.query['hub.verify_token'];
+    const challenge = req.query['hub.challenge'];
```

Comment:
Destructuring like that isn't possible because the messenger request returns data in this format: `"name":"messengerBot","hostname":"<redacted>","pid":73,"level":30,"msg":"{\"hub.mode\":\"subscribe\",\"hub.challenge\":\"<redacted>\",\"hub.verify_token\":\<redacted>"\"}","time":"2020-11-28T08:47:18.267Z","v":0}
`